freqtrade/optimize/custom_hyperopt.py

```diff
@@ -0,0 +1,153 @@
+# pragma pylint: disable=attribute-defined-outside-init
+
+"""
+This module load custom hyperopts
+"""
+import importlib
+import logging
+import os
+import sys
+from typing import Dict, Any, Callable
+
+from pandas import DataFrame
+
+from freqtrade.constants import Constants
+from freqtrade.optimize.interface import IHyperOpt
+
+sys.path.insert(0, r'../../user_data/hyperopts')
+
+logger = logging.getLogger(__name__)
+
+
+class CustomHyperOpt(object):
```

Comment:
Loading the python module should be handled the same way as in develop, see `freqtrade.strategy.resolver`